core/src/util/usFramework.cpp

```diff
@@ -72,7 +73,7 @@ void Framework::Start()
       d->operation = BundlePrivate::OP_ACTIVATING;
       break;
     case STATE_ACTIVE:
-      return;
+      break;
```

Comment:
I am not sure about the framework event. Looking at Knopflerfish and Felix, the former only sends the event once, but Felix sends it every time `start()` is called. Reading section 4.2.5 of the Core 6.0 specs:

> After the system bundle enters the ACTIVE state, a Framework STARTED event is broadcast.

it appears to me that the event should only be send once, *after* it enters the ACTIVE state. Calling `start()` again means we are already at that state. This is my interpretation. What passage were you refering to?